cohortextractor/tpp_backend.py

```diff
@@ -2032,6 +2032,122 @@ def patients_with_high_cost_drugs(
         GROUP BY t.Patient_ID
         """
 
+    def patients_with_ethnicity_from_sus(self, returning="code"):
+        returning_options = ["code", "group_6", "group_16"]
+        if returning is not None and returning not in returning_options:
```

Comment:
How come `returning` is allowed to be `None` here?

Reply:
Good question!  I _think_ I was planning to return either of our two groups alongside the raw code initially but then discussions made me realise we should always let the StudyDefinition author pick, I'll remove the None check as I think authors should always pick what they want… and that's how it's been coded.

Reply:
Fixed and rebased.